ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -196,27 +197,38 @@
 (defn- process-subscriptions
   "Process each subscription in subscriptions."
   [context subscriptions time-constraint]
-  (for [subscription subscriptions
-       :let [email-address (get-in subscription [:extra-fields :email-address])
-             coll-id (get-in subscription [:extra-fields :collection-concept-id])
-             query-string (-> (:metadata subscription)
-                              (json/decode)
-                              (get "Query"))
-             query-params (create-query-params query-string)
-             params1 (merge {"created-at" time-constraint}
-                            {"collection-concept-id" coll-id}
-                            query-params)
-             params2 (merge {"revision-date" time-constraint}
-                            {"collection-concept-id" coll-id}
-                            query-params)
-             gran-ref1 (search/find-granule-references context params1)
-             gran-ref2 (search/find-granule-references context params2)
-             gran-ref (distinct (concat gran-ref1 gran-ref2))]]
-      (if (seq gran-ref)
-        ;; These strings are just for debugging purpose. In another ticket
-        ;; I will figure out how to send email.
-        (str "Granules found for: " email-address)
-        (str "No granules found for: " email-address))))
+  (doseq [subscription subscriptions
+         :let [email-address (get-in subscription [:extra-fields :email-address])
+               coll-id (get-in subscription [:extra-fields :collection-concept-id])
+               query-string (-> (:metadata subscription)
+                                (json/decode)
```

Comment:
Change this to `(json/decode true)`, then you can use `:Query` to retrieve value on the next line. Using keyword over string is the CMR coding standard. This applies to the following lines (line 207 - 211) as well.

Reply:
Done.